src/Workspaces/Core/Portable/TaskList/TaskListItemDescriptor.cs

```diff
@@ -34,8 +41,25 @@ public static ImmutableArray<TaskListItemDescriptor> Parse(ImmutableArray<string
             {
                 if (item.Split(':') is [var token, var priorityString] &&
                     !string.IsNullOrWhiteSpace(token) &&
-                    int.TryParse(priorityString, NumberStyles.None, CultureInfo.InvariantCulture, out var priority))
+                    int.TryParse(priorityString, NumberStyles.None, CultureInfo.InvariantCulture, out var encoded))
                 {
+                    // From:
+                    // https://devdiv.visualstudio.com/DevDiv/_git/VS?path=/src/env/ErrorList/Pkg/Shims/TaskListOptions.cs&version=GBmain&line=133&lineEnd=134&lineStartColumn=1&lineEndColumn=98&lineStyle=plain&_a=contents
+#if false
```

Comment:
wonder if we should put this in a documentation comment so we can put it in a code block or something

Reply:
i'm fine any way :)